README.md

```diff
@@ -26,6 +27,12 @@ Libraries that are generally useful when working with Specs.
 
 ## Specs-powered projects
 
+## Templates
+
+Useful templates that let you get started more easily.
+
+* `shred-example` by @Marwes - (**Incomplete**) game template showing how to use Gluon scripting with Specs [![GitHub](https://img.shields.io/badge/repo-GitHub-green.svg)](https://github.com/Marwes/shred-example) ![incomplete](https://img.shields.io/badge/status-incomplete-blue.svg)
```

Comment:
No link here? https://github.com/Marwes/shred-example

Reply:
Yep, it's linked to from the badge

Reply:
Ah, I'm blind.